### PR TITLE
Feature/us13977 containerize nodejs

### PR DESCRIPTION
**Pull request by dillon-courts**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
 # crds-attendance
 Attendance Tracker
 
+This application is used to manually track attendance at crossroads sites and generate reports. It consists of an Angular2 frontend and a nodejs backend both w/in this repo. As of this writing the Angular2 frontend will be hosted on Netlify at attendancetracker.crossroads.net. The nodejs backend will be hosted in a container on a kubernetes cluster at api.crossroads.net/attenancetracker.
```
Comment (timmyg) [inline README.md]: that is a beautiful endpoint, other than the fact that attendance is spelled wrong

File: deployment/docker/docker-compose-local.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    build:
+      args:
+        - DEPLOYMENT_PATH
+      context: ../../
+      dockerfile: ./deployment/docker/prod.Dockerfile
+    container_name: crds-attendance
+    stdin_open: true
+    tty: true
+    env_file:
+      - .env
```
Comment (timmyg) [inline deployment/docker/docker-compose-local.yml]: is this used?

**Comment by timmyg** [comment]
something should be added in the readme to say what environment variables are needed to run the application (maybe just a link to https://github.com/crdschurch/crds-attendance/blob/62286d2bc4f97ec8fb0afbcce05e4ec93cde792e/deployment/kubernetes/deployment.yml#L24)

File: deployment/docker/docker-compose-local.yml (hunk 2 above)
Comment (dillon-courts) [reply deployment/docker/docker-compose-local.yml]: It is used if you are running locally... You would have to set up your own env file as I did not commit it due to sensitive info.

**Pull request merged**